renderer-macos/lib/Cocoa.re

```diff
@@ -249,6 +249,169 @@ module BriskTextView = {
   };
 };
 
+module BriskImage = {
+  type t = NSView.t;
+
+  type named = [
+    | `ImageActionTemplate
+    | `ImageAddTemplate
+    | `ImageAdvanced
+    | `ImageApplicationIcon
+    | `ImageBluetoothTemplate
+    | `ImageBonjour
+    | `ImageBookmarksTemplate
+    | `ImageCaution
+    | `ImageColorPanel
+    | `ImageColumnViewTemplate
+    | `ImageComputer
+    | `ImageEnterFullScreenTemplate
+    | `ImageEveryone
+    | `ImageExitFullScreenTemplate
+    | `ImageFlowViewTemplate
+    | `ImageFolder
+    | `ImageFolderBurnable
+    | `ImageFolderSmart
+    | `ImageFollowLinkFreestandingTemplate
+    | `ImageFontPanel
+    | `ImageGoLeftTemplate
+    | `ImageGoRightTemplate
+    | `ImageHomeTemplate
+    | `ImageIChatTheaterTemplate
+    | `ImageIconViewTemplate
+    | `ImageInfo
+    | `ImageInvalidDataFreestandingTemplate
+    | `ImageLeftFacingTriangleTemplate
+    | `ImageListViewTemplate
+    | `ImageLockLockedTemplate
+    | `ImageLockUnlockedTemplate
+    | `ImageMenuMixedStateTemplate
+    | `ImageMenuOnStateTemplate
+    | `ImageMobileMe
+    | `ImageMultipleDocuments
+    | `ImageNetwork
+    | `ImagePathTemplate
+    | `ImagePreferencesGeneral
+    | `ImageQuickLookTemplate
+    | `ImageRefreshFreestandingTemplate
+    | `ImageRefreshTemplate
+    | `ImageRemoveTemplate
+    | `ImageRevealFreestandingTemplate
+    | `ImageRightFacingTriangleTemplate
+    | `ImageShareTemplate
+    | `ImageSlideshowTemplate
+    | `ImageSmartBadgeTemplate
+    | `ImageStatusAvailable
+    | `ImageStatusNone
+    | `ImageStatusPartiallyAvailable
+    | `ImageStatusUnavailable
+    | `ImageStopProgressFreestandingTemplate
+    | `ImageStopProgressTemplate
+    | `ImageTrashEmpty
+    | `ImageTrashFull
+    | `ImageUser
+    | `ImageUserAccounts
+    | `ImageUserGroup
+    | `ImageUserGuest
+  ];
+
+  type source = [ | `file(string) | `named(string) | `bundle(string)];
```

Comment:
I think we should keep all kinds of variants constantly upper cased like most projects do. What if you don't like someone else's case preference 😋. Clash in what way?

Reply:
I don't know, let's table this for now. I'm not ready to give this up yet. 😄

Reply:
It's better we keep the Image prefix for named system images because there are like couple hundreds of them, which we will add later.

I don't want to see unrelated stuff in autocomplete. (VSCode sucks in this regard)

Reply:
in what way it sucks? I just think it's redundant. The fact that it "clashes" with anything else doesn't matter with poly variants